src/Pages/CategoryPage.tsx

```diff
@@ -9,17 +9,29 @@ function CategoryPage() {
   const { name } = useParams();
   const { items, categories, fetchCategoriesFromDb, fetchItemsFromDb } =
     useItems();
-
+  const [itemsFilterState, setItemsFilterState] = useState<any>(true);
```

Comment:
The type should be 'boolean' but ill approve the changes and maybe you can fix this later :)